locales/it/index.yml

```
@@ -1032,6 +1032,12 @@ identification:
       title: Identificazione richiesta
       sensorDescription: Tocca il sensore
       fallbackLabel: Usa il codice di sblocco
+  fail:
+    wrongCode: Codice errato
+    remainingAttempts: Ti rimangono {{attempts}} tentativi.
+    remainingAttemptSingle: Ti rimane {{attempts}} tentativo.
+    tooManyAttempts: Troppi tentativi di inserimento errati.
+    waitMessage: "Riprova tra:"
```

```suggestion
    waitMessage: "Chiudi l’applicazione o riprova tra:"
```
I guess this message could be more appropriate.
Isn't it?